src/GitHub.Api/SimpleApiClient.cs

```diff
@@ -66,7 +66,12 @@ public class SimpleApiClient : ISimpleApiClient
                     }
                 }
             }
-            // it'll throw if it's private
+            // it'll throw if it's private or an enterprise instance requiring authentication
+            catch (ForbiddenException)
+            {
+                if (!HostAddress.IsGitHubDotComUri(OriginalUrl.ToRepositoryUrl()))
```

Comment:
Flow analysis seems to think `OriginalUrl` could be `null`. However, it looks to me that's not legitimate. So maybe add an `Assert` here that it's not null?